src/instructlab/model/serve.py

```diff
@@ -121,6 +130,14 @@ def serve(
     from instructlab.model.backends import llama_cpp, vllm
 
     host, port = utils.split_hostport(ctx.obj.config.serve.host_port)
+
+    if tensor_parallel_size is not None:
+        if "--tensor-parallel-size" in ctx.args:
+            logger.info(
+                "'--gpus' flag used alongside '--tensor-parallel-size' in the vllm_args section of the config. Using the value found in the config"
```

Comment:
CLI args should always override a config file. Or am I misunderstanding the log message?